bin/shropshire-layer-update

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -e
+set -x
+
+source /data/mysociety/shlib/deployfns
+read_conf "$(dirname "$0")/../conf/general.yml"
+
+TMPDIR=$(mktemp -d) || exit 1
+trap 'rm -rf "$TMPDIR"' EXIT
+cd $TMPDIR
+
+declare -A types=(
+    ["ILUM"]='Iluminated_signs'
+    ["LAMP"]='Street_lights'
+    ["PLMP"]='Parish_Street_lights'
+)
+
+for type in ILUM LAMP PLMP; do
```

Comment:
I was wondering if it's possible to iterate over array keys in bash, to avoid repeating them. Turns out it is!

```bash
for type in "${!types[@]}"; do
```

Slightly arcane syntax, but might be a bit more future-proof this way?